### DISCUSSION TITLE
Copilot keeps suggesting a codefence at the end of many suggestions.

### DISCUSSION THREAD

**Discussion by TheColorRed** [community/community#112128] [Copilot Conversations]
### Select Topic Area

Bug

### Body

When using copilot with vscode, I keep getting suggestions with a code fence in all sorts of files `ts`, `html`, `scss`, `md`, `cpp`, etc.

Not sure why it keeps thinking I want a code fence at the end of the suggestion. Sometimes it's three `accent` characters, and sometimes its three `accent` characters followed by a language tag.

````
some suggestion....

```html
````

or 

````
some suggestion....

```
````

**Comment by Lanrad23** [comment]
I'd suggest poking around in the Copilot settings within VSCode a bit. There might not be a straightforward switch to flip to stop this from happening, but playing around with how aggressive or context-aware the suggestions are could potentially make a difference.

**Reply by Kaemikun** [reply]
agree

**Comment by Kaemikun** [comment]
It seems like there's a bit of confusion going on with the suggestions you're getting in Visual Studio Code. Instead of giving you helpful suggestions for completing your code, it's adding these unexpected code fences at the end. It's like suggesting a random ingredient for your recipe when you're trying to cook something entirely different!

To fix this, you can try a few things. First, check if any extensions you've installed might be causing this. Sometimes, extensions meant to help with coding can get a bit overeager and cause issues like this. Try disabling them one by one to see if the problem goes away.

Also, make sure you're working in the right language mode for the file you're editing. VS Code needs to know what kind of code you're writing to give you the right suggestions. If it's confused about the language, it might give you strange suggestions.

You might also want to take a look at your VS Code settings. Sometimes, there might be a setting that's causing this behavior. Look for anything related to code completion or snippets that could be causing the problem.

If all else fails, you can try updating VS Code to the latest version. Updates often come with bug fixes and improvements that might solve this issue.

And if you're still stuck, don't hesitate to reach out to the VS Code community for help. They're a friendly bunch and might have some more specific advice based on your situation!
